spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+# reserve for kind_details
+- _reserved: {from: 3555, to: 3575}
+- {layer: 'pois', kind: 'hostel'}
+- {layer: 'pois', kind: 'jewelry'}
+- {layer: 'pois', kind: 'karaoke'}
+- {layer: 'pois', kind: 'lawyer'}
+- {layer: 'pois', kind: 'midwife'}
+# reserve for kind_details
+- _reserved: {from: 3581, to: 3601}
+- {layer: 'pois', kind: 'mobile_phone'}
+- {layer: 'pois', kind: 'motorcycle'}
+- {layer: 'pois', kind: 'newsagent'}
+- {layer: 'pois', kind: 'newspaper'}
+- {layer: 'pois', kind: 'ngo'}
+- {layer: 'pois', kind: 'notary'}
+- {layer: 'pois', kind: 'optician'}
+- {layer: 'pois', kind: 'paediatrics'}
+# reserve for kind_details
+- _reserved: {from: 3610, to: 3630}
+- {layer: 'pois', kind: 'optometrist_paediatrics'}
+# reserve for kind_details
+- _reserved: {from: 3632, to: 3652}
+- {layer: 'pois', kind: 'perfumery'}
+- {layer: 'pois', kind: 'petting_zoo'}
+- {layer: 'pois', kind: 'physician'}
+# reserve for kind_details
+- _reserved: {from: 3656, to: 3676}
+- {layer: 'pois', kind: 'physiotherapist_physiotherapy'}
+# reserve for kind_details
+- _reserved: {from: 3678, to: 3698}
+- {layer: 'pois', kind: 'physiotherapist'}
+# reserve for kind_details
+- _reserved: {from: 3700, to: 3720}
+- {layer: 'pois', kind: 'playground'}
+- {layer: 'pois', kind: 'psychotherapist'}
+# reserve for kind_details
+- _reserved: {from: 3723, to: 3743}
+- {layer: 'pois', kind: 'podiatrist'}
+# reserve for kind_details
+- _reserved: {from: 3745, to: 3765}
+- {layer: 'pois', kind: 'political_party'}
+- {layer: 'pois', kind: 'rehabilitation'}
+# reserve for kind_details
+- _reserved: {from: 3768, to: 3788}
+- {layer: 'pois', kind: 'religion'}
+- {layer: 'pois', kind: 'research'}
+- {layer: 'pois', kind: 'sanitary_dump_station'}
+- {layer: 'pois', kind: 'scuba_diving'}
+- {layer: 'pois', kind: 'snowmobile'}
+- {layer: 'pois', kind: 'social_facility'}
+# reserve for kind_details
+- _reserved: {from: 3795, to: 3815}
+- {layer: 'pois', kind: 'speech_therapist'}
+# reserve for kind_details
+- _reserved: {from: 3817, to: 3837}
+- {layer: 'pois', kind: 'speech'}
+- {layer: 'pois', kind: 'stationery'}
+- {layer: 'pois', kind: 'studio'}
+# reserve for kind_details
+- _reserved: {from: 3841, to: 3861}
+- {layer: 'pois', kind: 'tax_advisor'}
+- {layer: 'pois', kind: 'telecommunication'}
+- {layer: 'pois', kind: 'therapist'}
+- {layer: 'pois', kind: 'tobacco'}
+- {layer: 'pois', kind: 'travel_agency'}
+- {layer: 'pois', kind: 'travel_agent'}
+- {layer: 'pois', kind: 'wildlife_park'}
+- {layer: 'transit', kind: 'station'}
+- {layer: 'transit', kind: 'stop_area'}
+- {layer: 'transit', kind: 'platform'}
+- {layer: 'buildings', kind: 'building'}
+# reserve for kind_details
+- _reserved: {from: 3873, to: 4073}
+- {layer: 'pois', kind: 'atv'}
+- {layer: 'pois', kind: 'bicycle_rental_station'}
+- {layer: 'pois', kind: 'boat_storage'}
+- {layer: 'pois', kind: 'bookmaker'}
+- {layer: 'pois', kind: 'bureau_de_change'}
+- {layer: 'pois', kind: 'car_parts'}
+- {layer: 'pois', kind: 'donation'}
```

Comment:
Remove `donation`, it was jumbled up in the health care funk.

Reply:
Fixed in cfe0df9.